Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'http'
 gem 'activesupport'
 gem 'json-jwt'
 gem 'jwt'
+gem 'dotenv-rails'
```

Comment:
since this is sinatra, not rails, is there a plain old `dotenv` gem we should use instead?